src/components/header/components/submenu-panel/styles.js

```diff
@@ -27,8 +34,19 @@ export const SubmenuWrapper = styled.div`
   }
 
   .submenu-wrapper {
-    max-width: 800px;
-    margin: auto;
+    width: 375px;
```

Comment:
Our min screen size we support is 320px (iphone 5/SE). We need to make sure this adapts to that width.